charts/datadog/templates/cluster-agent-deployment.yaml

```
@@ -240,13 +240,13 @@ spec:
           - name: DD_APM_INSTRUMENTATION_ENABLED
             value: "true"
```

I think this value should now return the value of `.Values.datadog.apm.instrumentation.enabled`

```suggestion
            value: {{ .Values.datadog.apm.instrumentation.enabled | quote }}
```